config.json.template

```diff
@@ -3,13 +3,19 @@
     "DATABASE_URL": "",
     "FROM_EMAIL": "",
     "EMAIL_SUBJECT": "Discord Verification",
-    "ROLE_NAME": "",
+    "VERIFIED_ROLE_NAME": "",
     "GUILD_ID": "",
     "WELCOME_CHANNEL_ID": "",
     "MEMBER_JOIN_MESSAGE": "",
     "EMAIL_REGEX": ".*@.*",
     "SMPT_JS_LOGIN_TOKEN": "",
     "CRYPTO_JSON_ALGORITHM": "aes256",
     "CRYPTO_JSON_ENCODING": "hex",
-    "CRYPTO_JSON_MEMBER_ENCRYPT_KEY": ""
+    "CRYPTO_JSON_MEMBER_ENCRYPT_KEY": "",
+    "SELF_JOIN_COMMAND": "!join",
+    "SELF_JOIN_REQUIRE_VERIFIED": true,
+    "SELF_JOIN_ROLE_NAMES": [
```

Comment:
can this become a comma separated string? 
Environment variables with nodes process.env system only allows for strings for key/values

Reply:
Latest commit now makes this config variable a comma separated string, with any spaces on either side of a comma being stripped automatically